cypress/support/fragments/settings/circulation/titleLevelRequests.js

```diff
@@ -42,8 +47,8 @@ export default {
           cy.expect(
             Checkbox({ name: 'titleLevelRequestsFeatureEnabled', disabled: false }).exists(),
           );
-          cy.do(TLRCheckbox.click());
-          cy.do(saveButton.click());
+          this.clickOnTLRCheckbox();
```

Comment:
Modifying of existing functions could lead to unexpected failures. Please do not use construnctions with 'this' keyword. 

As an alternative you can create report with execution of all tests where this method is used to be sure the this change did not affeted them.

Reply:
done (removed steps with 'this' statement)